content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -466,6 +466,391 @@ only this time for host `bookinfo.com` instead of `httpbin.example.com`.
         `"""`
     {{< /text >}}
 
+## Configure a TLS ingress gateway with secret discovery service enabled
+
+In this section you will configure a TLS ingress gateway that fetches credentials from ingress gateway agent via secret discovery service (SDS).
+The ingress gateway agent is running in the same pod as the ingress gateway, and watches credentials created in the same namespace as the ingress gateway.
+
+1.  Enable SDS at ingress gateway and deploy ingress gateway agent.
+    This feature is disabled by default, you need to manually enable the [feature flag]({{<github_blob>}}/install/kubernetes/helm/subcharts/gateways/values.yaml#L17) in helm,
```

Comment:
Can you use helm template to generate the yaml file?

Reply:
Done.